recipes/shap/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "shap" %}
+{% set version = "v0.22.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slundberg/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9cde030b3acb6bc334cfd4bde275ea7c72ae7c6d84cf90ba6fe1212b3686d6b6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - scipy
+    - scikit-learn
+    - matplotlib
+    - pandas
+    - tqdm
+    - ipython
+  run:
+    - python
+    - pandas
+    - numpy
+    - scikit-learn
+    - scipy
+    - matplotlib
+    - tqdm
+    - ipython
+test:
+  imports:
+    - shap 
+
+about:
+  home: https://github.com/slundberg/shap 
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A unified approach to explain the output of any machine learning model.'
+  description: |
+    SHAP (SHapley Additive exPlanations) is a unified approach to explain the output of any machine learning model. SHAP connects game theory with local explanations, uniting several previous methods and representing the only possible consistent and locally accurate additive feature attribution method based on expectations.
+  doc_url: https://shap.readthedocs.io/en/latest/
+  dev_url: https://github.com/slundberg/shap
+
+extra:
+  recipe-maintainers:
+    - mrandrewandrade 
+    - slundberg
```

Comment:
Can you confirm that the co-maintainers have agreed to be listed here? If not, please ping them to get a response.